ledger/error/src/main/scala/com/daml/error/ContextualizedErrorLogger.scala

```diff
@@ -20,6 +19,18 @@ trait ContextualizedErrorLogger {
   def warn(message: String, throwable: Throwable): Unit
   def error(message: String): Unit
   def error(message: String, throwable: Throwable): Unit
+
+  /** Formats the context as a string for logging */
+  protected[error] def formatContextAsString(contextMap: Map[String, String]): String = {
+    contextMap
+      .filter(_._2.nonEmpty)
+      .toSeq
+      .sortBy(_._1)
+      .map { case (k, v) =>
+        s"$k=$v"
+      }
+      .mkString(", ")
+  }
```

Comment:
I am not sure, but how about this:
```
    contextMap
      .toSeq
      .sortBy(_._1)
      .collect {
        case (key, value) if value.nonEmpty => s"$key=$value"
      }
      .mkString(", ")
```

Reply:
I just moved this function and would like to avoid changing it.